AutomatedTesting/Gem/PythonTests/Prefab/tests/duplicate_prefab/DuplicatePrefab_ContainingNestedEntitiesAndNestedPrefabs.py

```diff
@@ -0,0 +1,110 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+
+def DeletePrefab_ContainingNestedEntitiesAndNestedPrefabs():
+    """
+    Test description:
+    - Creates linear nested entities.
+    - Creates linear nested prefabs based of an entity with a physx collider.
+    - Creates a prefab from the nested entities and the nested prefabs.
+    - Duplicates the prefab.
+    - Checks that the prefab is correctly duplicated.
+    - Checks Undo/Redo operations.
+    """
+
+    from pathlib import Path
+    import copy
+
+    import azlmbr.legacy.general as general
+    import azlmbr.bus as bus
+    import azlmbr.entity as entity
+
+    from editor_python_test_tools.editor_entity_utils import EditorEntity
+    from editor_python_test_tools.prefab_utils import Prefab, get_all_entity_ids, wait_for_propagation
+    import Prefab.tests.PrefabTestUtils as prefab_test_utils
+
+    NESTED_ENTITIES_PREFAB_FILE_NAME = Path(__file__).stem + '_' + 'nested_entities_prefab'
+    NESTED_ENTITIES_NAME_PREFIX = 'Entity_'
+    NESTED_PREFABS_FILE_NAME_PREFIX = Path(__file__).stem + '_' + 'nested_prefabs_'
+    NESTED_PREFABS_NAME_PREFIX = 'NestedPrefabs_Prefab_'
```

Comment:
naming convention is different than other names used(avoid camelCase and capitalization)

Reply:
This is just for names of entities.

Reply:
I meant using 'NestedPrefabs_Prefab_' instead of 'nested_prefabs_prefab' . Which I thought was being used as a convention across tests. This is a nitpick. So I'd be okay with keeping it this way if it's common practice for naming in tests.

Reply:
Ah, okay. Yeah, this is pretty consistent to existing tests. This new test utilizes much of the same code from DeletePrefab_ContainingNestedEntitiesAndNestedPrefabs.py.